src/client/context.rs

```diff
@@ -557,6 +570,10 @@ impl Context {
         rest::delete_message(channel_id.into().0, message_id.into().0)
     }
 
+    /// Deletes all messages by Ids from the given vector in the given channel.
+    /// **Note**: This uses bulk delete endpoint which is not available
+    /// for user accounts.
+    /// Minimum amount of messages is 2 and maximum is 100.
```

Comment:
Let's put the "Minimum amount of messages" line above the note, and put a newline between each, like:


```rs
/// Deletes all mesages [...]
///
/// Minimum amount of messages is [...]
///
/// **Note**: This uses bulk [...]
```